### PR TITLE
login and signup implemented

### PR DESCRIPTION
**Pull request by govardhanreddyc79**
Implemented Login and SignUp pages for E-Com

### CODE REVIEW

File: Authentication/login.css [hunk 1]
```diff
@@ -0,0 +1,41 @@
+.login-container {
+    display: flex;
+    align-items: center;
+    flex-direction: column;
+    background-color:lightgrey;
```
Comment (aurobindoGupta) [inline Authentication/login.css]: use variable

**Review by aurobindoGupta — APPROVED** [review]
good work

File: Authentication/login.html [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+    <link href="https://ozone-ui.netlify.app/Components/Button/button.css" rel="stylesheet" type="text/css">
+    <link rel="stylesheet" href="../Authentication/login.css">
+</head>
+<body>
+    <nav class="nav space-around">
+        <div>
+            <a class="nav-header" href="">The Sports Adda</a>
+        </div>
+        <div>
+            <input type="text" placeholder="Find here" class="search">
```
Comment (laxmikanthkonthum) [inline Authentication/login.html:17]: label can be used as a best practice

File: Authentication/signup.html [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+            <input type="email" id="email" class="input-box" placeholder="xyz@gmail.com">
+            <label for="password">Password :</label>
+            <input type="password" id="password" class="input-box" placeholder="********">
+            
+            <div class="space-between margin-top">
+                <input type="checkbox" id="terms-conditions">
+                <span>
+                    <label for="terms-conditions">I accept all terms & conditions</label>
+                </span>
+                <!-- <span><a href="#" class="fgt-pwd"><small>Forgot your password ?</small></a></span> -->
```
Comment (laxmikanthkonthum) [inline Authentication/signup.html]: the commented part can be removed

**Review by laxmikanthkonthum — APPROVED** [review]
Added a couple of comments and rest looks good to me

File: Authentication/login.html (hunk 2 above)
Comment (govardhanreddyc79) [reply Authentication/login.html:17]: okay, Thanks

File: Authentication/login.css (hunk 1 above)
Comment (govardhanreddyc79) [reply Authentication/login.css]: sure, Thanks

File: Authentication/signup.html (hunk 3 above)
Comment (govardhanreddyc79) [reply Authentication/signup.html]: done

**Pull request merged**